### ISSUE TITLE
RPKM Correction 

### ISSUE THREAD

**Issue by ChadFibke** [jxu1234/MeDEStrand#1]
Hey @jxu1234,

I'm currently comparing MEDIPS to MeDEStrand, and was wondering if you could explain why your MeDEStrand.binMethyl.R does not correct for read depth and region size by calculating RPKM (this RPKM calculation is performed in MEDIPS on the signal /expected reads per bin values but is missing in MeDEStrand)?

Best,
Chad

**Comment by jxu1234** [comment]
Hi Chad,

Thank you for your email.
The reason I didn't build in functions to correct for read depth and region
size is because I found it does not provide additional improved accuracy. (
When I compared the inferred absolute methylation levels with gold standard
RRBS bin-wise methylation levels, add-on the additional correction or not. )
At that time, I believed that using a computational method to correct  read
depth bias from the MeDIP-Seq data itself doesn't work so well or at least
MEDIPS's way to make corrections is quite naive.
If you read 'MeDESTRAND's discussion session, there are more than 30+ to
correct read-depth related bias. I would choose the best one instead of
using MEDIPS's own. But I didn't work to further improve MeDEStrand since
my advisor didn't want me to and I started a new project then.

best,
jingting
